core/base/inc/TSystem.h

```diff
@@ -411,7 +411,7 @@ class TSystem : public TNamed {
    virtual Int_t           Exec(const char *shellcmd);
    virtual FILE           *OpenPipe(const char *command, const char *mode);
    virtual int             ClosePipe(FILE *pipe);
-   virtual TString         GetFromPipe(const char *command);
+   virtual TString         GetFromPipe(const char *command, int *ret = nullptr, bool redirectStderr = false);
```

Comment:
for consistency, I would use, if possible, kFALSE here.

Reply:
and also Int_t and Bool_t, or just changing false to kFALSE ? Thks.